packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -399,6 +401,10 @@ class FuchsiaDevice extends Device {
 
   @override
   Future<bool> stopApp(covariant FuchsiaApp app) async {
+    if (app == null) {
+      printError('Trying to stop a "null" app. Defaulting to success.');
```

Comment:
It seems like we shouldn't be calling this with a null app. Where is the call coming from?

If this should never be null, then we assert(app != null) for tests, and in release let the NSM propagate to crash logging.

If there's nothing user-actionable here, then this should probably be a `printTrace()`.

Reply:
When we run the driver tests, we try to stop any previously running applications. I changed this to assert and try to stop the application only if the package exists, let me know what you think.